tools/run_defacing_script.pl

```diff
@@ -439,10 +439,10 @@ sub check_if_deface_files_already_in_db {
     # base query
     my $query = "SELECT COUNT(*) "
                  . " FROM files f "
-                 . " JOIN mri_scan_type mst ON (mst.ID=f.AcquisitionProtocolID) ";
+                 . " JOIN mri_scan_type mst USING (mst.MriScanTypeID=f.MriScanTypeID) ";
```

Comment:
```suggestion
                 . " JOIN mri_scan_type mst ON (mst.MriScanTypeID=f.MriScanTypeID) ";
```

otherwise, get the following error:
```
DBD::mysql::st execute failed: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '.MriScanTypeID=f.MriScanTypeID)  WHERE (mst.MriScanTypeName = 't2-defaced' OR ms' at line 1 at tools/run_defacing_script.pl line 451.
```